.github/workflows/build.yaml

```diff
@@ -189,7 +189,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.8.0
+        uses: helm/kind-action@v1.10.0
```

Comment:
Bump action to use [kind 0.22](https://github.com/helm/kind-action/blob/v1.10.0/kind.sh#L21) which install [kubernetes 1.29](https://github.com/kubernetes-sigs/kind/releases/tag/v0.22.0) with [EOL 2025-02-28 ](https://kubernetes.io/releases/#release-v1-29)